### ISSUE TITLE
Cyrillic chars in lines.

### ISSUE THREAD

**Issue by nagolove** [amarakon/nvim-cmp-buffer-lines#15]
Hello! This is a great plugin for C programming. But it does not support lines with Cyrillic, only if I press `<C>-<X>-o` manually.

**Comment by amarz45** [comment]
You're right, I tried it and it doesn't work with lines with Cyrillic. I also tried it with Arabic characters and it also doesn't work. It seems like it only works on lines that only have Latin characters. I'm not sure what's causing this issue. I'm pretty sure it's an issue with nvim-cmp itself because I tried it with [cmp-buffer](https://github.com/hrsh7th/cmp-buffer) instead of cmp-buffer-lines and it still didn't work.

**Comment by nagolove** [comment]
Thanks for reply. I believe the situation will improve in the future.
